lib/src/utils.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:math';
```

Comment:
Would be good to have a more descriptive name on this file, if possible. (Not related to this line ofc)

Reply:
`options.dart` ?

Reply:
Options would seem like something to store global variables in, but these are just some specific helper functions, can they be grouped under some name that describes what they do?

Reply:
Well, they are global variables. However, it is difficult to foresee whether we'd want to add some more variables like this in the future, or maybe these are the only ones that will ever be.

Reply:
If there are others in the future they could (maybe even should) be in another file if they aren't related?
Maybe this could be named something related to the default angle or such?

Reply:
ok, i'll rename the file into `options/angles.dart`